### PR TITLE
f-content-cards@v2.1.0-beta.1

### PR DESCRIPTION
**Pull request by minrwhite**
The matching piece to complete the work from PR #312

Merging to a separate branch than master so as to keep master clean until we're certain this is working in the way we want to, while still being able to publish for consumption by other parties.

v2.1.0-beta.1
------------------------------
*October 5, 2020*

### Added
- `@custom-cards-callback` event dispatched on mount to expose a callback for consuming code to inject
  custom content cards

### Removed
- Card filtering/transforming/sorting logic moved to `f-metadata`; only the absolute limit number remains

### Changed
- Bumped dependency on f-metadata to version `3.0.0-beta.1`

---

## UI Review Checks

- [x] README and/or UI Documentation has been [created|updated]
- [x] Unit tests have been [created|updated]

### CODE REVIEW

**Comment by ashleynolan** [comment]
I'm just rebasing [this PR](https://github.com/justeat/fozzie-components/pull/293) so that we can get all the new components out of beta. If you give me 30 mins, should be able to get that reviewed so that you can just release this as a standard release after that?

**Comment by minrwhite** [comment]
@ashleynolan I'd still like to release this as a beta because of the changes it involves, but happy to do that on top of the typography changes once that's merged. Will mark this as do not merge, but still fish for comments/reviews for now.

**Review by asherstoppard — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

File: packages/f-content-cards/CHANGELOG.md
Comment (ashleynolan) [inline packages/f-content-cards/CHANGELOG.md:7]: Might be worth noting why this is a beta release – is it the link to `f-metadata` that's in beta?

File: packages/f-content-cards/src/components/ContentCards.vue
Comment (ashleynolan) [inline packages/f-content-cards/src/components/ContentCards.vue:122]: Only thing I'd be a bit wary of is naming these with respect to Braze – it makes sense now, but if we were ever to move away from Braze to another service in future, it would mean renaming all of these variables (just look at `appboy` 😆).

Not a crucial thing, but if there's a more generic name that makes sense for these I'd rename them to that instead.

File: packages/f-content-cards/CHANGELOG.md
Comment (minrwhite) [reply packages/f-content-cards/CHANGELOG.md:7]: Essentially yes - the semi decoupling between f-metadata and f-content-cards - hence why they're both `-beta` suffixed, for now. I'll add a message in.

File: packages/f-content-cards/src/components/ContentCards.vue
Comment (minrwhite) [reply packages/f-content-cards/src/components/ContentCards.vue:122]: That's fair - I could name it with respect to `metadata` as that's the module this is intended to adapt - the whole intent of the decoupling piece is so that f-content-cards can consume cards from other sources - initially just custom but maybe adding more marketing sources down the road. Will change references from `braze` to `metadata` in an upcoming PR if you're happy with that?

Comment (ashleynolan) [reply packages/f-content-cards/src/components/ContentCards.vue:122]: Yeh, makes sense. Happy for you to call it whatever you think makes the most sense 👍

**Pull request merged**
